cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/cli/RetrieveCommand.java

```diff
@@ -21,17 +20,11 @@
   @Option(name = {"-c", "--category"}, title = "Category", description = "Name of the feature category to retrieve. By default, all categories are searched")
   private String category;
 
-  @Option(name = {"-e", "--export"}, title = "Export", description = "Indicates whether the results should be exported. Defaults to false.")
-  private boolean export = false;
-
   @Option(name = {"-r", "--relevantSegments"}, title = "Relevant segments", description = "Comma separated list of segment IDs to which the query is to be limited.")
   private String relevantSegments;
 
   public void run() {
     final ContinuousRetrievalLogic retrieval = new ContinuousRetrievalLogic(Config.sharedConfig().getDatabase());
-    if (export) {
```

Comment:
Having this might be useful for testing and evaluation purposes, don't you think?

Reply:
Not really, i think for testing / evaluation, you'd want to use the APIs. For internal purposes, this was not really using a sensible abstraction (as evidenced by the fact that all evaluations of cineast / cottontail in past years have not used this functionality)

Reply:
ok, fair enough.